homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -174,6 +175,26 @@ def __init__(
             self._attr_supported_features |= CoverEntityFeature.STOP
         self._forced_resync = None
 
+    @property
+    def assumed_state(self) -> bool:
+        """If the device is hard wired we are polling state.
+
+        The hub will frequently provide the wrong state
+        for battery power devices so we set assumed
+        state in this case.
+        """
+        return not self._is_hard_wired
```

Comment:
Do you have a way we could make this user optional - I personally hate the assumed state and want to trust HA (we do all our moving through there) so I want shades to look closed in the UI when they are so

Reply:
You can override this with [customize](https://www.home-assistant.io/docs/configuration/customizing-devices/)

Reply:
> You can override this with [customize](https://www.home-assistant.io/docs/configuration/customizing-devices/#manual-customization) There is an example on how to force `assumed_state` to `false`.

We should set assume to false by default and tell the complainers to override in the UI 😂 
Hate appeasing the minority groups

Reply:
> Hate appeasing the minority groups

It would make it easier, but it really is an assumed state since we don't poll the shade so its hard to justify presenting it any other way.

Many Hunter Douglas / Powerview / etc will actually report the state on change someday, but I'm really not holding my breath on that one.